src/main/java/TaskManager.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class TaskManager {
+    public static void main(String[] args) {
+        Task[] taskList = new Task[3];
+        taskList[0] = new Task("read book");
+        taskList[1] = new Task("return book");
+        taskList[2] = new Task("buy bread");
+
+        taskList[0].markAsDone();
```

Comment:
Would be better if you are able to continuously taking in inputs and adding them to tasks instead of having only 3 inputs